packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -467,87 +468,87 @@ function OrderBasketItemActions({
     setItems([
       ...items,
       {
-        uuid: medication.uuid,
-        display: medication.drug?.display,
-        previousOrder: null,
-        startDate: new Date(),
         action: 'RENEW',
-        drug: medication.drug,
+        asNeeded: medication.asNeeded,
+        asNeededCondition: medication.asNeededCondition,
+        careSetting: medication.careSetting?.uuid,
+        commonMedicationName: medication.drug?.display,
+        dateActivated: new Date(),
```

Comment:
This is the key change, renaming this property from `startDate` to `dateActivated`.